vector/src/main/java/im/vector/util/DinsicUtils.java

```diff
@@ -0,0 +1,36 @@
+package im.vector.util;
+
+import java.util.List;
+
+import im.vector.activity.LoginActivity;
+
+/**
+ * Created by cloud on 1/22/18.
+ */
+
+public class DinsicUtils {
+    /**
+     * Tells if an email is from gov
+     *
+     * @param email the email address to test.
+     * @return true if the email address is from gov
+     */
+    public  static boolean isFromFrenchGov(final String email) {
+        return (null != email && email.toLowerCase().endsWith (LoginActivity.AGENT_OR_INTERNAL_SECURED_EMAIL_HOST));
+    }
```

Comment:
AGENT_OR_INTERNAL_SECURED_EMAIL_HOST should be declared in this class not in LoginActivity.